packages/dapp/src/components/Tokens/Tokens.tsx

```diff
@@ -1,18 +1,48 @@
 import * as React from "react";
+import { connect } from "react-redux";
 import styled from "styled-components";
+import { State } from "../../redux/reducers";
 import { Helmet } from "react-helmet";
+import { EthAddress } from "@joincivil/core";
+import { getFormattedEthAddress } from "@joincivil/utils";
 
 import { UserTokenAccount } from "@joincivil/components";
 import ScrollToTopOnMount from "../utility/ScrollToTop";
 
-export const Tokens: React.SFC = props => {
+export interface TokensProps {
+  userAccount: EthAddress;
+}
+
+export const TokensComponent: React.SFC<TokensProps> = props => {
   return (
     <>
       <Helmet>
         <title>Token Account - The Civil Registry</title>
       </Helmet>
       <ScrollToTopOnMount />
-      <UserTokenAccount />
+      <UserTokenAccount
+        userLoggedIn={true}
+        userTutorialComplete={false}
+        userAccount={props.userAccount}
+        supportEmailAddress={"support@civil.co"}
+        faqUrl={"https://cvlconsensys.zendesk.com/hc/en-us"}
+        foundationAddress={"0xf1176B0aeb7914B5472B61c97A4CF0E0bcacB579"}
```

Comment:
Is this the appellate multisig? Should this come from some utils thing that specifies RInkeby vs Mainnet vs Ganache so it can be tested in diff environments w/o affecting mainnet?

Reply:
We currently use the same address in all environments and Airswap switches between rinkeby and mainnet I will add those to utils though.